frontend/www/js/omegaup/components/problem/CollectionProblem.vue

```diff
@@ -1,21 +1,19 @@
 <template>
-  <div class="col-sm-4 mt-2">
-    <div class="border border-dark">
-      <div class="row">
-        <div class="col-sm-4 d-flex align-items-center justify-content-end">
+  <div class="card mb-3" style="width: 290px">
```

Comment:
```suggestion
  <div class="card mb-3" omegaup-collection-problem>
```

idealmente no deberían haber `style` inline. en la sección de estilo puedes hacer

```scss
[omegaup-collection-problem] {
  width: 290px;
  margin-right: 0.5em;
}
```